test/cql-pytest/conftest.py

```diff
@@ -167,3 +170,34 @@ def random_seed():
 
 # TODO: use new_test_table and "yield from" to make shared test_table
 # fixtures with some common schemas.
+
+# To run the Scylla tools, we need to run Scylla executable itself, so we
+# need to find the path of the executable that was used to run Scylla for
+# this test. We do this by trying to find a local process which is listening
+# to the address and port to which our our CQL connection is connected.
+# If such a process exists, we verify that it is Scylla, and return the
+# executable's path. If we can't find the Scylla executable we use
+# pytest.skip() to skip tests relying on this executable.
+@pytest.fixture(scope="module")
```

Comment:
Small nitpick: this can have "session" scope, instead of "module" - there is no reason to re-run it again when switching test files.

Reply:
I'll send another version, I like V8s anyway.